nucypher/datastore/datastore.py

```diff
@@ -84,7 +84,7 @@ class Datastore:
 
     # LMDB has a `map_size` arg that caps the total size of the database.
     # We can set this arbitrarily high (1TB) to prevent any run-time crashes.
-    LMDB_MAP_SIZE = 1_000_000_000_000
+    LMDB_MAP_SIZE = 1_000_000
```

Comment:
Is it the size in bytes? So it's only 1Mb now? Why such a massive change? 

Also if it's intentional, the comment should be changed too.

Reply:
I hear ya - to be fair this is in a commit all by itself with the message "Use small LMDB_MAP_SIZE to get CI to run".

If I do not change this value, CI will fail.  What is the best way to handle this considering the very well defined scope of this PR?

Reply:
Considering the scope of this PR, this commit perhaps doesn't belong here at all :) Can we set the value based on an environment variable?